app/contributing/conditional-rendering.md

```diff
@@ -109,19 +111,23 @@ The filter expects new lines before and after `if_version` or `if_plugin_version
 
 The above will be rendered as a single table.
 
-### Inline content
+### Whitespace control
+
+Both `if_version` and `if_plugin_version` use the same implementation as Liquid's `if` tag. Therefore, they have the same [whitespace control](https://shopify.github.io/liquid/basics/whitespace/).
 
-If you want to conditionally render content in a sentence, you can use `if_version` or `if_plugin_version`
-and specify `inline:true`:
+When rendering lists with `{if_version`, `if_plugin_version`, or `if`, there's a caveat: a hyphen needs to be added to the right side of the tags (for consistency, it could be either side) for them to be rendered correctly.
```

Comment:
```suggestion
When rendering lists with `if_version`, `if_plugin_version`, or `if`, there's a caveat: you must add a hyphen to the right side of the tags (for consistency, it could be either side) to render them correctly.
```